drivers/ieee802154/ieee802154_nrf5.c

```diff
@@ -97,7 +97,8 @@ static void nrf5_rx_thread(void *arg1, void *arg2, void *arg3)
 		 * The last 2 bytes contain LQI or FCS, depending if
 		 * automatic CRC handling is enabled or not, respectively.
 		 */
-#if defined(CONFIG_IEEE802154_NRF5_RAW)
+#if defined(CONFIG_IEEE802154_NRF5_RAW) || \
+	defined(CONFIG_NET_L2_OPENTHREAD)
```

Comment:
Do we need to update the rest of 15.4 drivers with something similar to this?

Reply:
Let's see first how the whole behave with nrf5.

I am not entirely sure I'll want to keep this that way in the future (actually, I am not found of CONFIG_IEEE802154_*_RAW stuff either).